### PR TITLE
Added error handling of required fields in search

### PR DESCRIPTION
**Pull request by dyampols**
- Added required star labels for user input of parameters 
- Errors will now appear if user did not input a field

### CODE REVIEW

File: travvy/src/components/Home.vue [hunk 1]
```diff
@@ -12,7 +12,18 @@
...
+    <p v-if="errors.length" class="error">
+      <b>Please enter the following missing information:</b>
+      <!-- this will bind all the errors together after the form is completed and return them to the user-->
+      <u1 id="checkForm" class="red"> 
+        <br>
+        <li v-for="error in errors" :key="error"> {{ error }}</li>
+        <!-- v-bind="error in errors"-->
+      </u1>
+    </p>
+    <label for="Destination">Select a Destination*:</label>
```
Comment (lkdfount) [inline travvy/src/components/Home.vue:26]: Good idea to use the *, it will keep the site more consistent

**Review by lkdfount — APPROVED** [review]

File: travvy/src/components/Home.vue (hunk 1 above)
Comment (dyampols) [reply travvy/src/components/Home.vue:26]: Thanks!

Comment (emilynairnn) [inline travvy/src/components/Home.vue:26]: Looks like you used some of the error handling code from sign up. Great to keep it consistent throughout the site and ensure when errors are returned to users, they are displayed in the same way throughout the site. Nice work!

**Review by emilynairnn — APPROVED** [review]
Looks good to me!

File: travvy/src/components/Home.vue (hunk 1 above)
Comment (dyampols) [reply travvy/src/components/Home.vue:26]: Thanks for figuring out how to implement it!

Comment (lucasungaretti) [reply travvy/src/components/Home.vue:26]: Agreed!

**Review by lucasungaretti — APPROVED** [review]
LGTM good job!

**Pull request merged**